code-samples/popular-topics/permissions/index.js

```diff
@@ -0,0 +1,130 @@
+const util = require('util');
+const { Client, Permissions } = require('discord.js');
+const client = new Client();
+
+client.on('ready', () => {
+	console.log('Ready!');
+});
+
+client.on('message', message => {
```

Comment:
The code style for this file is fairly inconsistent with the rest of the files in the guide. Check out [this file](https://github.com/discordjs/guide/blob/master/code-samples/creating-your-bot/commands-with-user-input/index.js) to see a decent example of what I mean and try to apply it to this file.

Reply:
Tried to apply, if you want it to look differently let me know

Reply:
The blank lines between the if/else if statements should be removed